core/app/models/mno_enterprise/alert.rb

```diff
@@ -2,5 +2,22 @@ module MnoEnterprise
   class Alert < BaseResource
     property :created_at, type: :time
     property :updated_at, type: :time
+    property :kpi_id
```

Comment:
Should the `kpi_id` be exposed? I thought we were getting rid of the foreign keys

Reply:
It should for now, but will be addressed later in https://github.com/maestrano/mno-enterprise/pull/560